frontend/src/tests/components/Commons/AdminDisplayTable.test.js

```diff
@@ -0,0 +1,90 @@
+
+import { fireEvent, render, waitFor } from "@testing-library/react";
+import  adminDisplayFixtures  from "fixtures/adminDisplayFixtures";
+import AdminDisplayTable from "main/components/Commons/AdminDisplayTable"
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+import { currentUserFixtures } from "fixtures/currentUserFixtures";
+
+
+describe("AdminDsplayTable tests", () => {
+  const queryClient = new QueryClient();
+  
+  test("renders without crashing for empty table with user not logged in", () => {
+    const currentUser = null;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <AdminDisplayTable admins={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+
+    );
+  });
+  
+  test("renders without crashing for empty table for ordinary user", () => {
+    const currentUser = currentUserFixtures.userOnly;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <AdminDisplayTable admins={[]} currentUser={currentUser} />
```

Comment:
Again, these are not "admins".  Admins are people that have administrator access.  

This might be a display table for admins to look at something, but what they are looking is not a list of "admins".  Naming is super important to keeping the code understandable for future programmers.

Reply:
Got it.